tests/images.spec.ts

```diff
@@ -18,3 +24,43 @@ test("search for custom image and create an instance from it", async ({
 
   await deleteInstance(page, customInstance);
 });
+
+test("Export an image", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  const imageAlias = await createImageFromInstance(page, instance);
```

Comment:
```suggestion
  // creating an image from an instance, because we can only export unified images
  // and can't test the export with a standard image fetched from the image server
  // we can remove this step once export of split images is enabled by the backend
  const instance = randomInstanceName();
  await createInstance(page, instance);
  const imageAlias = await createImageFromInstance(page, instance);
```
Maybe explain why this step is needed, so we can remember to clean it up later, once we can export standard images fetched from the image server.